### ISSUE TITLE
Our process section

### ISSUE THREAD

**Issue by mantagen** [dwyl/dwyl-site#100]
Possible examples of design for the list (Understand, Build, Review).

![image](https://cloud.githubusercontent.com/assets/12934669/12923659/7653f5ae-cf4e-11e5-9bb1-0e6737ad1ab3.png)

![image](https://cloud.githubusercontent.com/assets/12934669/12923686/8ab431b2-cf4e-11e5-94f2-57f0bd38230e.png)

![image](https://cloud.githubusercontent.com/assets/12934669/12923713/a5e44fb2-cf4e-11e5-9bf6-3aa598275c52.png)

**Comment by hdrdavies** [comment]
I feel the 'Our Process' Section is possibly a little busy at the moment and it's not totally clear where the user should look... 

We should look for feedback from users on this.

**Comment by iteles** [comment]
I think we've lost some of the important copy in this section, especially "By iterating over these 3 simple steps we maintain a tight feedback loop between the people building and using the product/service, ensuring we meet your needs quickly and without waste."  It's not an obvious statement.

<img width="1778" alt="screen shot 2016-02-10 at 23 05 54" src="https://cloud.githubusercontent.com/assets/4185328/12964984/f4395dfe-d04a-11e5-9220-5c9693f80a7a.png">

**Comment by iteles** [comment]
I also appreciate what we're trying to do with 'Understand', 'Build', 'Review' but I don't think those make enough sense on their own to an audience who's not accustomed to the process. They may need a couple of words extra.
e.g. '1. Understand the problem'

**Comment by iteles** [comment]
Based on #119, this diagram may not work the way we expected.

**Question: Can we have the writing on the left seeing as that is clearer than the diagram?**

@nelsonic The best I've come up with so far for the illustration is _this_ and it isn't ideal:
Traditionally, applications were built from start to finish based on the assumption that every feature of the original idea was equally important to your users. This typically leads to long projects where you don't know until the very end if your application will work for your audience as you intended.     
[top illustration]
Our agile approach lets you test incremental features with your users at regular short intervals and ensure you focus your time and effort on what's really important to your audience, building in your learnings from actual usage of your application.     
[MVP approach]

**Comment by hdrdavies** [comment]
This would give a lot more context to the diagram and help users understand the message - I like it. What makes you think it isn't ideal?

**Comment by iteles** [comment]
Too wordy. And I don't think starting on the 'traditional' way makes it clear enough that that's _not_ how we do it...

**Comment by iteles** [comment]
@nelsonic *bump*

**Comment by nelsonic** [comment]
@iteles thanks for the *bump*. I think we need to *test* & *iterate* on our own copy. not focussing too much on the words at this stage. More focussed on how the *overall* page makes me feel. 
> "*Can I trust these kids...?*"

**Comment by markwilliamfirth** [comment]
Closing due to new redesign
